app/main.py

```diff
@@ -1 +1,59 @@
-#  write your code here
+class Animal:
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> str:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} "
+                  f"food points...")
+            self.is_hungry = False
+            return self.appetite
+        else:
+            return 0
+
+
+class Cat(Animal):
+    def __init__(
+            self,
+            name: str,
+            is_hungry: bool = True
+    ) -> None:
+        super().__init__(
+            name,
+            appetite=3,
+            is_hungry=is_hungry
+        )
+
+    def catch_mouse(self) -> str:
+        print("The hunt began!")
+
+
+class Dog(Animal):
+    def __init__(
+            self,
+            name: str,
+            is_hungry:
+            bool = True
```

Comment:
Place `bool = True` on the same line with `is_hungry` to match its type annotation. You could indicate in your type annotation that this argument is optional as well throughout your code where it is used.